pandas/core/arrays/string_arrow.py

```diff
@@ -35,121 +31,27 @@
 )
 from pandas.core.arraylike import OpsMixin
 from pandas.core.arrays.base import ExtensionArray
+from pandas.core.arrays.string_ import StringDtype
 from pandas.core.indexers import (
     check_array_indexer,
     validate_indices,
 )
 from pandas.core.missing import get_fill_func
 
-try:
-    import pyarrow as pa
-except ImportError:
-    pa = None
```

Comment:
We indeed don't need to lazily import pyarrow anymore for the sake of the dtype object. But if we want to expose the ArrowStringArray in `pandas.core.arrays` like all other arrays, we are still going to need to keep this behind an ImportError .. 

(not sure if we want to expose ArrowStringArray, but until that's decided I would maybe leave those imports as is)